docs/source/intro/run_docker.rst

```diff
@@ -15,23 +15,85 @@ This image contains a fully pre-configured AiiDA environment which makes it part
    :gutter: 3
 
-   .. grid-item-card:: Install Docker on your workstation or laptop
+   .. grid-item-card:: Install Docker on your PC
 
-      To install Docker, please refer to the `official documentation <https://docs.docker.com/get-docker/>`__.
+      Docker is available for Windows, Mac and Linux and can be installed in different ways.
 
-   .. grid-item-card:: Start container and use AiiDA interactively
+      .. tab-set::
 
-      Start the container with (replace ``latest`` with the version you want to use, check the `Docker Hub <https://hub.docker.com/r/aiidateam/aiida-core-with-services/tags>`__ for available tags/versions):
+         .. tab-item:: Colima on MacOS
 
-      .. parsed-literal::
+            `Colima <https://github.com/abiosoft/colima>`_ is a new open-source project that makes it easy to run Docker on MacOS.
+            It is a lightweight alternative to Docker Engine with a focus on simplicity and performance.
 
-         $ docker run -it aiidateam/aiida-core-with-services:latest bash
+            If you need multiple Docker environments, Colima is the recommended way.
+            With colima, you can have multiple Docker environments running at the same time, each with its own Docker daemon and resource allocation thus avoiding conflicts.
 
-      You can specify a name for the container with the ``--name`` option for easier reference later on:
+            To install the colima, on MacOS run:
 
-      .. parsed-literal::
+            .. parsed-literal::
 
-         $ docker run -it --name aiida-container aiidateam/aiida-core-with-services:latest bash
+               $ brew install colima
+
+            Or check Check `here <https://github.com/abiosoft/colima/blob/main/docs/INSTALL.md>`__ for other installation options.
+
+            After installation, start the docker daemon by:
+
+            .. parsed-literal::
+
+               $ colima start
+
+         .. tab-item:: Docker CE on Linux
+
+            The bare minimum to run Docker on Linux is to install the `Docker Engine <https://docs.docker.com/engine/install/>`_.
+            If you don't need a graphical user interface, this is the recommended way to install Docker.
+
+            .. note::
+
+               You will need `root` privileges to perform the `post-installation steps <https://docs.docker.com/engine/install/linux-postinstall/>`_.
+               Otherwise, you will need to use `sudo` for every Docker command.
+
+
+
+   .. grid-item-card:: Start/stop container and use AiiDA interactively
+
+      Start the image with the `docker command line (docker CLI) <https://docs.docker.com/engine/reference/commandline/cli/>`_.
+
+      The ``latest`` tag is the image with the most recent stable version of ``aiida-core`` installed in the container.
+      You can replace the ``latest`` tag with the version you want to use, check the `Docker Hub <https://hub.docker.com/r/aiidateam/aiida-core-with-services/tags>`_ for available tags.
+
+      .. tab-set::
```

Comment:
Initially there is instruction for docker desktop, and it should be added for windows. I remove it simply because I can not test it.

Maybe we keep it as a placeholder?